boards/st/steval_stwinbx1/board.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/usb/usb_device.h>
+#include <zephyr/drivers/uart.h>
+
+#if defined(CONFIG_CONSOLE) && defined(CONFIG_UART_CONSOLE)
+#if DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_console), zephyr_cdc_acm_uart)
+/*
+ * Enable console on USB CDC_ACM
+ */
+static int steval_stwinbx1_usb_console_init(void)
+{
+	const struct device *const dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+
+	if (!device_is_ready(dev) || usb_enable(NULL)) {
+		return -1;
```

Comment:
use Return values as specified 
_-ENOENT – If device was not found - or isn’t a deferred one.
-errno – For other errors._

Reply:
> [!NOTE]  
>  should be fixed here too
> https://github.com/zephyrproject-rtos/zephyr/blob/d6d3c1098d5a46edaaa1f1e42133e5bb10698765/boards/st/sensortile_box_pro/board.c#L23-L25

Reply:
PR done for sensortile_box_pro: https://github.com/zephyrproject-rtos/zephyr/pull/72748